### PR TITLE
Devil affairs

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

Remakes https://github.com/fulpstation/fulpstation/pull/563

## Why It's Good For The Game

"cheap gamemode at a reasonable price?", "spendid."

### CODE REVIEW

File: fulp_modules/features/antagonists/devil_affairs/devil_affairs.dm [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+/// Affairs Agents can only roll Internal or External, rather than the normal factions.
+/datum/antagonist/traitor/devil_affairs/pick_employer(faction)
+	employer = ROLE_DEVIL_AFFAIRS
+	traitor_flavor = strings(DEVILAFFAIR_FLAVOR_FILE, employer, directory = "fulp_modules/strings")
+
+/datum/antagonist/traitor/devil_affairs/greet()
+	. = ..()
+	var/crime = pick(
+		"distribution of contraband",
+		"unauthorized erotic action on duty",
```
Comment (OneAsianTortoise) [inline fulp_modules/features/antagonists/devil_affairs/devil_affairs.dm:46]: I know it is just flavor text or something, but are you sure about using erotic reference or stuff idk.

File: fulp_modules/features/antagonists/devil_affairs/devil_pinpointer.dm [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/atom/movable/screen/alert/status_effect/agent_pinpointer/devil_affairs
+	name = "Internal Affairs Integrated Pinpointer"
```
[the remark this answers is not in the document]
Comment (JohnFulpWillard) [reply fulp_modules/features/antagonists/devil_affairs/devil_pinpointer.dm:2]: was meant to mean internal as in, its your internal pinpointer, but tbh it doesnt make much sense

**Comment by Pepsilawn** [comment]
U 4got to run prettier

**Comment by OneAsianTortoise** [comment]
what why

**Pull request closed without merging**